doc/modules/changes/20210708_fildes.txt

```diff
@@ -0,0 +1,4 @@
+New: There is now a ‘static’ option for the temperature field that is set-up similarly to the ‘static’ option for compositional fields. This allows the temperature field to be static while advection is on so you can still advect and build up elastic stresses.
```

Comment:
Please remove the .txt ending of this file ( `git mv doc/modules/changes/20210708_fildes.txt  doc/modules/changes/20210708_fildes`)